requirements.txt

```diff
@@ -28,7 +28,7 @@ Flask-Mail==0.9.1
 Flask-Migrate==4.0.4
 Flask-Sitemap==0.4.0
 Flask-SQLAlchemy==3.0.5
-Flask-WTF==1.1.1
+Flask-WTF==1.1.2
```

Comment:
https://github.com/wtforms/flask-wtf/releases/tag/v1.1.2
```
Fixed Flask 2.3 deprecations of werkzeug.urls.url_encode and
flask.Markup https://github.com/wtforms/flask-wtf/pull/565 https://github.com/wtforms/flask-wtf/issues/561
Stop support for python 3.7 https://github.com/wtforms/flask-wtf/pull/574
Use pyproject.toml instead of setup.cfg https://github.com/wtforms/flask-wtf/pull/576
Fixed nested blueprint CSRF exemption https://github.com/wtforms/flask-wtf/pull/572
```